### PR TITLE
Use SymEngine in Crypto

### PR DESCRIPTION
**Pull request by ShikharJ**

### CODE REVIEW

**Comment by ShikharJ** [comment]
Ping @isuruf.

File: sympy/crypto/crypto.py [hunk 1]
```diff
@@ -10,14 +10,14 @@
 
 from string import whitespace, ascii_uppercase as uppercase, printable
 
-from sympy import nextprime
-from sympy.core import Rational, Symbol
-from sympy.core.numbers import igcdex, mod_inverse
+from sympy import Symbol
```
Comment (isuruf) [inline sympy/crypto/crypto.py]: As in the previous PR, this is not good

File: sympy/crypto/crypto.py [hunk 2]
```diff
@@ -10,14 +10,14 @@
...
+from sympy.core.backend import (Rational, totient, gcd, isprime, igcdex,
+                                mod_inverse, nextprime)
 from sympy.core.compatibility import range
 from sympy.matrices import Matrix
```
Comment (isuruf) [inline sympy/crypto/crypto.py]: Same here

**Comment by gxyd** [comment]
@ShikharJ can you please make the requested changes.

**Comment by ShikharJ** [comment]
@gxyd I've got my exams in the coming week. I'll get back to this after they're over.

**Comment by gxyd** [comment]
Ah, that is okay. Good luck with exams.

**Comment by ShikharJ** [comment]
Same as [here](https://github.com/sympy/sympy/pull/13214#issuecomment-533908620).

**Pull request closed without merging**